Appl/Breadbox/BbxBrow/local.mk

```diff
@@ -22,7 +22,8 @@ GOCFLAGS += $(COMPILE_OPTIONS)
 # -Os: favor size of execution speed
 # JavaScript code uses #if rather than #ifdef
 #XCCOMFLAGS = -d -dc -Z -Os -O $(COMPILE_OPTIONS:S|JAVASCRIPT_SUPPORT|JAVASCRIPT_SUPPORT=1|g)
-XCCOMFLAGS = -zu -zc $(COMPILE_OPTIONS:S|JAVASCRIPT_SUPPORT|JAVASCRIPT_SUPPORT=1|g)
+XCCOMFLAGS = -zu $(COMPILE_OPTIONS:S|JAVASCRIPT_SUPPORT|JAVASCRIPT_SUPPORT=1|g)
```

Comment:
Is this some documented issue?

Reply:
I think this is the same issue you fixed in 42a15ce9c42aa109a944c27c282b77e2eab479d2, where `-zc` is not compatible with changing codesegments. This is the same reason why we used `#pragma option -dc-` for Borland C, but unfortunately `-zc` cannot be disabled locally through a pragma in Watcom.